lib/broccoli/ember-app.js

```diff
@@ -1598,6 +1595,9 @@ EmberApp.prototype._contentForConfigModule = function(content, config) {
   @param {Object} config
 */
 EmberApp.prototype._contentForAppBoot = function(content, config) {
+  content.push('var prefix = \'' + config.modulePrefix + '\';');
```

Comment:
Previously this was inside of a function, this would now show up in global scope.